deploy/kubernetes/delete.sh

```
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+set -e
+
+if [ "$EUID" -eq 0 ]
+  then echo "Run without sudo/root"
+  exit
+fi
+
+NC='\033[0m'
+RED='\033[0;31m'
+
+delete () {
+    echo "Deleting sock-shop namespace resources"
+    kubectl delete all --all -n sock-shop
+    kubectl delete namespace sock-shop || true
```

What does the `|| true` do?
If the delete operation fail, why would we want to suppress the error?